src/osp/Resource/Package.h

```diff
@@ -136,7 +136,7 @@ class Package
         std::map<std::string, Resource<TYPE_T>, std::less<>> m_resources;
     };
 
-    ResPrefix_t get_prefix() { return m_prefix; }
+    constexpr ResPrefix_t const& get_prefix() { return m_prefix; }
```

Comment:
constexpr functions aren't, inherently, const. You'll want to add the trailing const keyword as well.